src/Resources.resx

```diff
@@ -129,13 +129,15 @@
 	</data>
 
 	<data name="BI0002" xml:space="preserve">
-		<value>Could not compile the API bindings.</value>
+		<value> Could not compile the API bindings. 
+		{0}
+		</value>
 		<comment>
 		</comment>
 	</data>
 
 	<data name="BI0026" xml:space="preserve">
-		<value>Could not parse the command line argument '{0}': {1}</value>
+		<value>Could not parse the command line argument '--warnaserror': {0}</value>
```

Comment:
not sure it's a good change
it makes it too easy for someone to translate `--warnaserror` by mistake
and the same error message could be used for something else (later) and would require updating translations

Reply:
Okay I can change it back. I changed it because that's the message that was being used in btouch.cs, which is different from how it's documented in generator-errors.md.

<img width="2115" alt="Screen Shot 2019-12-04 at 1 54 26 PM" src="https://user-images.githubusercontent.com/54864665/70171980-cccec180-169d-11ea-9d44-07a054ca6c91.png">